### ISSUE TITLE
Suppress ‘Lutris is starting…’ dialog when running from a terminal

### ISSUE THREAD

**Issue by solbero** [lutris/lutris#3436]
As a part of the redesign in v5.8.0 a ‘Lutris is starting…’ dialog was added when starting Lutris. I can see that this dialog was added so the user knows that something is happening while Lutris is setting up and initialising.

However, when running Lutris from the terminal this dialogue is redundant, since all the information is outputted to the terminal anyway for the user to monitor. Also, it is sometimes in the way since it usually pops up over the terminal window.

I wonder if it would be possible to suppress this dialog when launching Lutris through the terminal and also when using Lutris with terminal commands such as `lutris lutris:rungameid/`? 

![Screenshot_2021-02-21_13-30-09](https://user-images.githubusercontent.com/7732292/108625126-61f7da80-7449-11eb-9802-950357a9c235.png)

**Comment by strycore** [comment]
This is a horrible idea.

**Comment by solbero** [comment]
I could have been clearer in my initial post why I thought about this in the first place and submitted this issue. It related to the [kodi-lutris-addon](https://github.com/RobLoach/lutris-kodi-addon).

The addon request the games managed by Lutris using the command `lutris --list-games --installed --json` to build the folder structure and list items in Kodi and get the information necessary (such as the ID and slug) to launch games. This results in the ‘Lutris is starting…’ dialog being shown briefly on screen above the Kodi interface when the addon is opened.

This could be confusing for the users of the addon since it seems like Lutris is starting, when what the addon is doing is getting the list of managed games from Lutris and using it internally.

My idea when I opened this issue was that not showing the ‘Lutris is starting…’ dialog for only the command mentioned above would be quite an exception to implement, and therefore I suggested not showing it at all when interacting with Lutris through a terminal.

Based on your response I understand that this is not something that fits the goals of the project.
